src/components/TopBar/TopBar.tsx

```diff
@@ -31,27 +33,39 @@ const TopBar: React.FC<TopBarProps> = () => {
           <StyledLogoWrapper>
             <Logo />
           </StyledLogoWrapper>
-          {mobileMenu ? (
-            <StyledMobileMenuWrapper>
-              <Nav onDismiss={handleDismissMobileMenu} mobileMenu={mobileMenu}/>
-            </StyledMobileMenuWrapper>
-          ) : (
-            <StyledNavWrapper>
-              <Nav onDismiss={handleDismissMobileMenu} mobileMenu={mobileMenu}/>
-            </StyledNavWrapper>
+          { location.pathname !== '/hometest' && (
+            <>
+            {mobileMenu ? (
+              <StyledMobileMenuWrapper>
+                <Nav onDismiss={handleDismissMobileMenu} mobileMenu={mobileMenu}/>
+              </StyledMobileMenuWrapper>
+            ) : (
+              <StyledNavWrapper>
+                <Nav onDismiss={handleDismissMobileMenu} mobileMenu={mobileMenu}/>
+              </StyledNavWrapper>
+            )}</>
           )}
+          <Spacer />
           <StyledLeftMenuBalancesWrapper>
             <StyledAccountButtonWrapper>
               <StyledTopBarDarkModeSwitch>
                 <DarkModeSwitch />
```

Comment:
we want to have that on the bottom of this page on smaller devices, similar to how it shows on other pages we have currently